tests/integration/executor/test_graph_functions.py

```
@@ -0,0 +1,313 @@
+        gf.create_node(["Person", "Employee"], name="Alice")
+        gf.create_node(["Person"], name="Bob")
+        gf.create_node(["Customer"], name="Charlie")
+
+        # Get all nodes and their labels
+        results = gf.execute("MATCH (n) RETURN n.name AS name, labels(n) AS labels ORDER BY n.name")
+        assert len(results) == 3
+
+        # Check that Alice has Employee label
+        alice = results[0]
+        assert alice["name"].value == "Alice"
+        alice_labels = [l.value for l in alice["labels"].value]
```

Ruff (0.14.14)

[error] Ambiguous variable name: `l`

(E741)